modular_skyrat/modules/customization/datums/components/crafting/recipes.dm

```diff
@@ -13,7 +13,7 @@
 	subcategory = CAT_MISCFOOD
 
 /datum/crafting_recipe/food/neep_tatty_haggis
-	name = "Haggis with neeps and tatties"
+	name = "Haggis With Neeps and Tatties"
```

Comment:
I feel like this is what you meant to do? I'm a bit confused here, to be fair.
```suggestion
	name = "Haggis with Neeps and Tatties"
```

Reply:
Crafting recipes are all capitalised, but generally you wouldn't capitalise conjunctions or short (1-3) letter prepositions.